From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Tue, 15 Dec 2015 22:04:07 +0100
Subject: [PATCH] properly detect shm on kfreebsd / hurd

The rt could be the default. The thing with ac_try_compile is that passes
because we have the headers but linking fails due missing headers.
ac_try_link would be better but then we need to know the lib :)
This should work on kfreebsd and gnu hurd (which has currently gnu0.7
as target_os set).
	           
Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---

diff --git a/configure.ac b/configure.ac
index 66dbc7e..d48c458 100644
--- a/configure.ac
+++ b/configure.ac
@@ -821,6 +821,12 @@ case "$target_os" in
      linux*)
         EXTRALIBS="$EXTRALIBS -lrt"
      ;;
+     kfreebsd*)
+        EXTRALIBS="$EXTRALIBS -lrt"
+     ;;
+     gnu*)
+        EXTRALIBS="$EXTRALIBS -lrt"
+     ;;
      *)
 esac
 fi
